src/types/save.service.abstract.ts

```diff
@@ -0,0 +1,7 @@
+import {Observable} from 'rxjs/Observable';
```

Comment:
This class is not necessary. it should be removed, and instead the user should just `implement` the NgxLoggerHttpService